tests/test.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once 'php.php';
+
+class CreateTableTest extends TestCase
+{
+    public function testCreateTableData(): void
+    {
+        $mockData = [
+            [
+                'id' => 1,
+                'name' => 'Red Ochre',
+                'color' => 'Red',
+                'HEX' => '#FF0000',
+                'Geology' => 'Iron Oxide',
+                'image_closeup' => 'closeup.jpg',
+                'image_site' => 'site.jpg',
+                'country' => 'USA',
+                'town' => 'Sedona',
+                'coordinateslat' => '34.8697',
+                'coordinateslong' => '-111.7608'
+            ]
+        ];
+
+        $expectedOutput = '<tr>';
+        $expectedOutput .= '<td>1</td>';
+        $expectedOutput .= '<td>Red Ochre</td>';
+        $expectedOutput .= '<td>Red</td>';
+        $expectedOutput .= '<td>#FF0000</td>';
+        $expectedOutput .= '<td>Iron Oxide</td>';
+        $expectedOutput .= '<td><img src="closeup.jpg" alt="Closeup view"></td>';
+        $expectedOutput .= '<td><img src="site.jpg" alt="Site view"></td>';
+        $expectedOutput .= '<td>USA</td>';
+        $expectedOutput .= '<td>Sedona</td>';
+        $expectedOutput .= '<td>34.8697</td>';
+        $expectedOutput .= '<td>-111.7608</td>';
+        $expectedOutput .= '</tr>';
```

Comment:
I think this could be refactored to not repeat $expectedOutput on every line (I can show you and example.)